hibernate-ehcache/src/main/java/org/hibernate/cache/ehcache/EhCacheRegionFactory.java

```diff
@@ -66,11 +67,13 @@ public void start(SessionFactoryOptions settings, Properties properties) throws
 			}
 			if ( configurationResourceName == null || configurationResourceName.length() == 0 ) {
 				final Configuration configuration = ConfigurationFactory.parseConfiguration();
+				setCacheManagerNameIfNeeded( settings, configuration );
```

Comment:
Is this right... or should this method instead be using newInstance() and returning the same cache manager?

Reply:
You talk about line 71 now, yes?
```java
manager = new CacheManager( configuration );
```

My memory somewhat fails me here, but I think one `SessionFactory` instance is bound to one `CacheManager` instance. I don't know we'd want to share ever… but maybe I'm wrong. 
I _could_ change that, I don't think it'd be a big deal… previously throwing config would now reuse an existing `CacheManager` instance if one'd be available. Could make sense. I let you @chrisdennis make the call… You're the man in charge here ;)

Reply:
Okay - that makes sense. The last thing I have to say is: https://jira.terracotta.org/jira/browse/EHC-885

Reply:
Some providers do allow multiple SessionFactories to "bind" to a single cache instance.  This is actually why the notion of a "region name prefix" was added.

Up to you guys if you want to support that.

Reply:
I really don't *want* to support anything on the 2.x line any more ;)

Reply:
`hibernate-ehcache` currently pulls in `net.sf.ehcache:ehcache:2.10.3` while `hibernate-jcache` pulls in `org.ehcache:ehcache:3.0.0` (although only as a test dep).

Did you guys want to update that to only use 3?

Reply:
My understanding is that the last place for all this was "lets try and make Ehcache 3 not need it's own provider".  I don't think thats changed, which leaves the hibernate-ehcache in this weird place. @ljacomet care to join this party with an opinion?

Reply:
I'll check wrt EHC-885 this weekend and see how we can have this work as part of this. 
Expect the thing to be final by Monday. wrt 3.x et al… I'll let the Terracotta folks call the shot :)

Reply:
FWIW `hibernate-infinispan` got "relocated" to the Infinispan project for all of the same reasons.

We could do a similar relocation for Ehcache, basically dropping support for Ehcache 2